apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/DataSetQueue.java

```diff
@@ -137,16 +138,24 @@ public void updatePendingDataSets() {
    */
   private void addManifestToList(
       DataSetManifest.DataSetManifestId manifestId, String manifestS3Key) {
-    DataSetManifest manifest;
-
     /*
-     * If the keyspace we're scanning doesnt exist, bail early (This can happen if we're loading synthetic data,
+     * If the keyspace we're scanning doesnt exist, bail early (This can happen if
+     * we're loading synthetic data,
      * as it checks the regular incoming folder for the manifest first.)
      */
-    if (!s3TaskManager.getS3Client().doesObjectExist(options.getS3BucketName(), manifestS3Key)) {
+    HeadObjectRequest headObjectRequest =
+        HeadObjectRequest.builder().bucket(options.getS3BucketName()).key(manifestS3Key).build();
+    try {
+      s3TaskManager.getS3Client().headObject(headObjectRequest);
+    } catch (NoSuchKeyException | NoSuchBucketException e) {
+      LOGGER.warn(
+          "Unable to find keyspace {} in bucket {} while scanning for manifests.",
+          manifestS3Key,
+          options.getS3BucketName());
```

Comment:
We weren't loggin this before.  Why log it now as a warning?

Reply:
I switched this to a debug statement - I think it's valuable info and I *know* I didn't write this original log statement, so it did exist in the codebase at one point in time. I think having it as a debug is okay? If you feel strongly about it I can remove it, though.